### PR TITLE
Add benchmark

### PR DESCRIPTION
**Pull request by simonsso**

### CODE REVIEW

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -280,7 +280,7 @@ impl InstanceFilter<Call> for ProxyType {
             ProxyType::NonTransfer => !matches!(
                 c,
                 Call::Balances(..)
-                    | Call::Vesting(pallet_vesting::Call::vested_transfer(..))
+                    // | Call::Vesting(pallet_vesting::Call::vested_transfer(..))
```
Comment (simonsso) [inline runtime/src/lib.rs:283]: @Gauthamastro  Should maybe orml vesting connect to here somehow?

Comment (Gauthamastro) [reply runtime/src/lib.rs:283]: Wait for my pr

Comment (simonsso) [reply runtime/src/lib.rs:283]: Good

**Comment by simonsso** [comment]
@Gauthamastro  This one!!!

**Pull request closed without merging**